straxen/contexts.py

```diff
@@ -8,6 +8,7 @@
 import typing as ty
 from pandas.util._decorators import deprecate_kwarg
 import socket
+from straxen.plugins.peaklets.peaklet_classification_som import PeakletClassificationSOM
```

Comment:
Please conform with the existing coding style for plugin management.
plugins are not imported explicitly.